metadata/v1.2/0/0-move-stdlib.json

```diff
@@ -0,0 +1,7 @@
+{
+    "title": "Update aptos framework to cb4ba0a57c998c60cbab65af31a64875d2588ca5",
+    "description": "Update move stdlib package to the version compiled from git hash cb4ba0a57c998c60cbab65af31a64875d2588ca5 at bytecode format v5.",
+    "source_code_url": "https://github.com/aptos-labs/mainnet-proposals/blob/34fc783919ae4d58c42237a994702f0b06fbebc7/sources/v1.2/0/0-move-stdlib.move",
+    "discussion_url": "https://aptos.dev"
```

Comment:
Let's use this one for discussion https://github.com/aptos-labs/aptos-core/issues/6557